File: app/src/main/java/com/example/yunlibrary/common/activity/java/YunDemoActivity.java
package com.example.yunlibrary.common.activity.java;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.yunlibrary.R;
import com.example.yunlog.kotlin.YunLog;

public class YunDemoActivity  extends AppCompatActivity {

    private TextView textView;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_demo_layout);
        textView = findViewById(R.id.textview_print_log);
        textView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                printLog();
            }
        });
    }

    private void printLog() {
        YunLog.INSTANCE.i(null,"Kotlin : call from Java, Print I level log");
    }
}

File: app/src/main/java/com/example/yunlibrary/common/activity/kotlin/YunDemoActivity.kt
package com.example.yunlibrary.common.activity.kotlin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.yunlibrary.R
import com.example.yunlog.kotlin.YunLog
import kotlinx.android.synthetic.main.activity_demo_layout.*
typealias JavaYunLog = com.example.yunlog.java.YunLog

class YunDemoActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_demo_layout)
        textview_print_log.setOnClickListener {
            printLog()
        }
    }

    fun printLog() {
        YunLog.i(null, "Kotlin : print the I level log")
        JavaYunLog.i(arrayListOf("Java : print the I level log"))
        JavaYunLog.i("Java TAG", "Java : print the I Level Log")
    }
}